onnx/reference/ops/op_optional_get_element.py

```diff
@@ -9,9 +9,7 @@
 class OptionalGetElement(OpRun):
     def _run(self, x):  # type: ignore
         if x is None:
-            return ([],)
-        if isinstance(x, list):
+            return (None,)
```

Comment:
I think the best implementation would be to fail (raise an exception) if `x is None` and `return x` otherwise.

Of course, this is dependent on the overall conventions used in the reference implementation. I agree with your comments in https://github.com/onnx/onnx/pull/4815 that we don't use any wrappers. A value is either None or one of the other valid values, such as a tensor or list of tensors.

Reply:
done